contrib/agentsvg/README.md

```diff
@@ -0,0 +1,23 @@
+# CFEngine agent SVG / logo generator
+
```

Comment:
Seems like the README.md should include ` ![cf-agent, the CFEngine mascot](agent.png)`

Reply:
Done.